SU-HSDO/suhumsci
# SHS-5568: Allow editors to select style on Accordion component

# READY FOR REVIEW

## Summary
Adds `Style` select to allow editors to apply the `hb-accordion-secondary` class to Accordion components

## Steps to Test
1. Create a new flexible page
2. Add two Accordion components
3. On one of the two components set the `Style` select to `Secondary`
4. Save the new page
5. The Accordion component with the `Style` select set to `Secondary` should now be using the `Secondary` style
6. The Accordion component with the `Style` select set to `Default` should use the default style

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
